@xen-orchestra/backups/_runners/_writers/IncrementalRemoteWriter.mjs

```diff
@@ -143,8 +143,10 @@ export class IncrementalRemoteWriter extends MixinRemoteWriter(AbstractIncrement
 
     let metadataContent = await this._isAlreadyTransferred(timestamp)
     if (metadataContent !== undefined) {
-      // @todo : should skip backup while being vigilant to not stuck the forked stream
+      // skip backup while being vigilant to not stuck the forked stream
       Task.info('This backup has already been transfered')
+      Object.values(deltaExport.streams, stream => stream.destroy())
```

Comment:
Are the streams correctly destroyed ? I think Object.values() only takes one parameter and does not apply a function

Reply:
deltaExport.streams is an object , where the keys are the vhd id, and the value is the stream of this vhd

Reply:
nice catch